electrical/pcb/README.md

```diff
@@ -89,19 +89,25 @@ Make sure to attach D1 in the correct orientation!
 
 Insert a fuse (part# B1) into the fuse holder (as shown in Figure 3.8 below)
 
-Using the appropriate matching XT30 connectors, connect the power headers to a power supply (or your battery), the external switch (part# B9), and the multimeter (part# B10)
+Using the appropriate matching XT30 connectors, connect the power headers to a power supply (or your battery), the main external power switch (part# B9), and the multimeter (part# B10)
 
-Use this diagram to determine to wire things together:
+Use these diagrams to determine how to wire things together:
 
 TODO: add diagram of board / power supply / external switch / multimeter wiring
```

Comment:
```suggestion
```
I believe your image addresses this TODO, correct?

Reply:
Yes, I believe you right. These look good.